src/__tests__/__snapshots__/FilterListItem.js.snap

```diff
@@ -34,8 +34,210 @@ exports[`FilterList.Item renders the given "is" prop 1`] = `
   font-weight: 600;
 }
 
-<a
+<b
   className="c0"
+  theme={
```

Comment:
I have no idea why the changes to `FilterList.Item` are adding the theme back into the markup for snapshots 🤔 I'm not seeing it show up in the DOM. cc @mxstbr

Reply:
That's really strange 🤔

Reply:
Yeah that's why 👍